### PR TITLE
deploying the latest version of unlock-js for local dev

### PR DESCRIPTION
**Pull request by julien51**
# Description

Making sure all dev environment now use the latest version on unlock-js

# Checklist:

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [X] This PR only contains configuration changes (package.json, etc.)
  - [ ] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [ ] My code follows the style guidelines of this project, including naming conventions
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [X] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: tickets/scripts/deploy-unlock.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ let deployedLockAddress
 
 serverIsUp(1000 /* every second */, 120 /* up to 2 minutes */)
   .then(() => {
-    return deploy(host, port, 'v02', newContractInstance => {
```
Comment (julien51) [inline tickets/scripts/deploy-unlock.js]: tickets was even on an older version :/

**Comment by julien51** [comment]
We should eventually clean things up so that we only use a single script to deploy the contracts!

**Comment by cellog** [comment]
We could export a string `currentUnlockVersion` and use that in all the deploy scripts

**Comment by julien51** [comment]
it's worth talking about this... but in another PR?

**Comment by julien51** [comment]
That integration test failure is weird. Re-running again.

**Comment by julien51** [comment]
Ok, same error. No idea what is going on here.

```

**Comment by akeem** [comment]
> We should eventually clean things up so that we only use a single script to deploy the contracts!

I'd like to cut over to the docker compose setup when we have time

**Pull request merged**